package.json

```diff
@@ -9,29 +9,33 @@
     "postinstall": "scripts/copy-katex-assets && scripts/install-twemoji-assets",
     "check": "scripts/check",
     "format": "npx prettier '**/*.@(ts|js|svelte|json|css|html)' --ignore-path .gitignore --write",
-    "test:unit": "TZ='UTC' vitest run",
-    "test:components": "cypress run --component",
-    "test:e2e": "cypress run",
```

Comment:
Could we leave a
```
"test:e2e": "playwright test --quiet"
```
Seems like something I would do frequently.

Reply:
What I actually would like is to reduce these logs by refactoring the app. So that we can disable some parts that aren't being tested. For example, we should be able to disable Ethereum features if they're not being tested, which would probably remove all the logs.

The reason I'd like to leave them "on" in dev mode is that we should notice when the app is trying to access resources that aren't stubbed or if there are console logs/warnings/errors, so that we address them right away.

Reply:
Ah okay got it 👍 
I would still prefer to keep some npm script to execute quickly the e2e test suite. Maybe can do a `"test:e2e": "playwright test"`

Reply:
a8efbd7869ec9a96ed35cd1ee268757eda1f205c